### ISSUE TITLE
Transfer, Modify, Copy, Move Filter

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Animeimp/slmiv#12]
```
A way to filter my inventory to just show me all my items
that are transferable [or Modify, Copy, Move]
```

Original issue reported on code.google.com by `joesox` on 14 Jan 2009 at 6:00

**Comment by GoogleCodeExporter** [comment]
```
reported by  sircaren on Yahoo! group
```

Original comment by `joesox` on 14 Jan 2009 at 6:00

**Comment by GoogleCodeExporter** [comment]
```
'Create Word Hit List': Lists highest to lowest of word frequencies found in the
current treeview inventory
FIXED in version 2.10.16
```

Original comment by `joesox` on 23 Jan 2009 at 7:46
- Changed state: **Verified**
